package.json

```diff
@@ -2,33 +2,40 @@
   "name": "NeoLink",
   "version": "0.1.29",
   "description": "A Chrome Extension Wallet for the Neo Smart Economy",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CityOfZion/NeoLink.git"
+  },
   "scripts": {
-    "build": "node utils/build.js",
+    "build": "NODE_ENV=production node utils/build.js",
     "start": "node utils/webserver.js"
   },
   "devDependencies": {
-    "babel-core": "^6.3.15",
+    "babel-core": "^6.26.0",
     "babel-eslint": "^7.1.0",
-    "babel-loader": "^6.2.0",
+    "babel-loader": "^7.1.2",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-decorators-legacy": "^1.2.0",
     "babel-plugin-transform-runtime": "^6.5.2",
-    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.3.13",
-    "babel-preset-react-hmre": "^1.0.0",
+    "babel-preset-react-hmre": "^1.1.1",
     "babel-preset-react-optimize": "^1.0.1",
     "babel-preset-stage-0": "^6.3.13",
     "babel-runtime": "^6.3.19",
-    "css-loader": "^0.25.0",
-    "file-loader": "^0.11.2",
+    "css-loader": "^0.28.7",
+    "file-loader": "^1.1.6",
     "fs-extra": "^0.30.0",
-    "html-loader": "^0.4.5",
-    "html-webpack-plugin": "2.24.1",
+    "html-loader": "^0.5.1",
+    "html-webpack-plugin": "^2.24.1",
     "postcss-loader": "^2.0.9",
     "preact-material-components": "^1.3.2",
-    "style-loader": "^0.13.0",
-    "webpack": "2.2.1",
-    "webpack-dev-server": "^2.3.0",
+    "react": "^16.2.0",
```

Comment:
Why do we need React in dev? Shouldn't we either exclusively use Preact or React?

Reply:
It is required as a peer dependency by some modules we use. If it's not there, we get 5 warnings in the build console. I figure it's better to suppress the warnings to keep any noise out of the build than to just ignore them.